README.md

```diff
@@ -26,15 +26,17 @@ Interactive version of the block clock component and secondary widgets. Goal:
 
 It's helpful to build out an application step-by-step, optimizing each one. This is just a starting point and can be changed as we go along based on user needs.
 
+![Cross-platform layouts](/images/cross-platform-layouts.png)
+
 ![Bitcoin Core App major versions](/images/bitcoin-core-app-versions.png)
 
 ### Version 1.0: Node runner
 
-Initial goal is to get more people to run nodes on smartphones, by offering a simple, elegant experience. The UI is focused on a highly visual block clock. Functionality is limited, but users should be able to connect other applications to the node.
+Initial goal is to set the foundation for a modern UI based on best practices, and to get more people to run nodes across operating systems and device types by offering a simple, elegant experience. The UI is focused on a highly visual block clock. Functionality is limited, but users should be able to connect other applications to the node.
```

Comment:
```suggestion
Initial goal is to set the foundation for a modern UI based on best practices, and to get more people to run nodes across operating systems and device types by offering a simple, elegant experience. The UI is focused on a highly visual block clock. Functionality is limited.
```

Atleast for now, remove this. For now i'd rather not think about interactions with other applications. This stage is just a node runner.

In terms of interacting with other applications, this would be done the way it is currently done with core; connecting through the RPC interface. 

Introducing other forms of connecting to the underlying bitcoind instance is currently out of scope.

Reply:
ACK